doc/man1/openssl-ocsp.pod.in

```diff
@@ -76,10 +82,12 @@ This command performs many common OCSP tasks. It can be used
 to print out requests and responses, create requests and send queries
 to an OCSP responder and behave like a mini OCSP server itself.
 
-=head1 OPTIONS
-
 This command operates as either a client or a server.
-The options are described below, divided into those two modes.
+This first entry in the L</SYNOPSIS> section lists the client options,
```

Comment:
s/This first/The first/g

Reply:
I don't think you need this link, but you can make it point to the head2 sections, no?

Reply:
My intention was not so much to give a reference, but rather to point out that there are *two* entries in the SYNOPSIS section, a fact which might otherwise be overlooked. Do have an alternative suggestion how to formulate it?

Reply:
I don't think you need it.  But if you do, then keep it.

Reply:
Oh you are right. It was necessary, before the idea to add the `=head2` subsections came to my mind. Now it's not needed anymore.

Reply:
Commit be2b7a52c7788798202a646ffec6ed598f98b57c restores the original formulation on master.